src/braket/pennylane_plugin/braket_device.py

```diff
@@ -124,7 +126,8 @@ def _pl_to_braket_circuit(self, circuit, **run_kwargs):
             **run_kwargs,
         )
         for observable in circuit.observables:
-            braket_circuit.add_result_type(translate_result_type(observable))
+            dev_wires = self.map_wires(observable.wires).tolist()
```

Comment:
Just to make sure: `braket_circuit` would be awaiting consecutive wire ordering, correct? That's why we need to map the custom device wires using `map_wires`.